src/hb-aat-layout-feat-table.hh

```diff
@@ -32,13 +32,28 @@
 
 namespace AAT {
 
-struct FeatureName {
+struct SettingName
+{
+  inline bool sanitize (hb_sanitize_context_t *c) const
+  {
+    TRACE_SANITIZE (this);
+    return_trace (c->check_struct (this));
+  }
 
+  protected:
+  HBUINT16	setting;	/* The setting. */
+  NameID	nameIndex;	/* The name table index for the setting's name. */
```

Comment:
The spec says `nameIndex` is an `int16` but ours is unsigned. It doesn't matter eventually anyway but just saying.